04week/higherOrder.js

```diff
@@ -2,17 +2,39 @@
 
 const assert = require('assert');
 
-function forEach(arr, callback) {
-  // Your code here
+
+const numArray = [2, 4, 7, 8];
+//Create a forEach() function that takes an array of items and a function that runs the function //arr.length number of times.
+
+function forEach(arr) {
```

Comment:
This function should take an array as it's first argument and a function as it's second. For every item in the array (LOOP THROUGH), preform the function argument on each item.